### PR TITLE
[4.0] Fixes issue #23746- unclear sidebar

### PR DESCRIPTION
**Pull request by bahl24**
Pull Request for Issue #23746 

### Summary of Changes
Increasing padding of sidebar menus solves the issue.

### Actual result
screenshot from 2019-02-03 00-33-09

### Documentation Changes Required
None

### CODE REVIEW

**Comment by C-Lodder** [comment]
If you're trying to create spacing between each element, I'd suggest using a bottom margin instead, e.g:
```
margin-bottom: 1rem; 
```

**Comment by bahl24** [comment]
@C-Lodder Incorporated your suggetions. Kindly give it a look

**Comment by C-Lodder** [comment]
@bahl24 Please remove the padding-bottom on the line above and use `rem` units for the margin value.

**Comment by C-Lodder** [comment]
@wilsonge What happened to Hound?

**Comment by bahl24** [comment]
@infograf768 Kindly give it a look

**Review by C-Lodder — APPROVED** [review]

File: administrator/templates/atum/scss/blocks/_sidebar-nav.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@
   }
 
   li {
-    padding-bottom: 3px;
+    margin-bottom: 0.7rem;
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_sidebar-nav.scss]:
```suggestion
    margin-bottom: .7rem;
```

**Comment by bahl24** [comment]
@Quy Incorporated your suggetion

**Comment by bahl24** [comment]
@Quy Kindly review it.

**Comment by Quy** [comment]
On my screen `Users` is out of view so you have to scroll up. Maybe try `.5rem`.

**Comment by bahl24** [comment]
> On my screen `Users` is out of view so you have to scroll up. Maybe try `.5rem`.

@Quy Kindly check.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on a84ddc4eb33de47e2209298fb2304e957ccab731<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23747">issues.joomla.org/tracker/joomla-cms/23747</a>.</sub>

**Comment by bahl24** [comment]
@C-Lodder Kindly test

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on a84ddc4eb33de47e2209298fb2304e957ccab731<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23747">issues.joomla.org/tracker/joomla-cms/23747</a>.</sub>

**Comment by bahl24** [comment]
@infograf768 Kindly test if the PR is RTC

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23747">issues.joomla.org/tracker/joomla-cms/23747</a>.</sub>

**Comment by bahl24** [comment]
@Quy @infograf768 Kindly merge this pr, if there are no issues

**Comment by wilsonge** [comment]
Thanks!

**Comment by bahl24** [comment]
Thanks @wilsonge

**Pull request merged**